### PR TITLE
Convert final files to Kotlin

### PR DESCRIPTION
**Pull request by marianeum**
+ [x] Check this if you have read and followed the [contributing guidelines](https://github.com/Skyscanner/backpack-android/blob/main/CONTRIBUTING.md)

Remember to include the following changes:
+ [x] Component `README.md`
+ [x] Tests
+ [x] Docs (either update [backpack-docs](https://github.com/Skyscanner/backpack-docs) now, or create a follow up ticket)

If you are curious about how we review, please read through the [code review guidelines](https://github.com/Skyscanner/backpack/blob/main/CODE_REVIEW_GUIDELINES.md)

### CODE REVIEW

File: Backpack/src/main/java/net/skyscanner/backpack/toast/BpkToast.kt [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+import net.skyscanner.backpack.text.BpkFontSpan
+import net.skyscanner.backpack.text.BpkText
+import net.skyscanner.backpack.text.BpkText.Companion.getFont
+import net.skyscanner.backpack.text.BpkText.FontDefinition
+
+object BpkToast {
+  const val LENGTH_SHORT = Toast.LENGTH_SHORT
+  const val LENGTH_LONG = Toast.LENGTH_LONG
+
+  @JvmStatic
```
Comment (marianeum) [inline Backpack/src/main/java/net/skyscanner/backpack/toast/BpkToast.kt:35]: This makes sure people can access it the same way they could before

File: android-configuration.gradle [hunk 2]
```diff
@@ -41,6 +41,10 @@ android {
     warningsAsErrors true
     disable 'UnusedResources' // we're exposing resources for consumers
   }
+
+  buildFeatures {
```
Comment (marianeum) [inline android-configuration.gradle:45]: This stops generating a `buildConfig` file, which was also generated in java and we don't need it.

File: backpack-compose/build.gradle.kts [hunk 3]
```diff
@@ -31,6 +31,7 @@ apply(from = "${rootProject.projectDir}/android-configuration.gradle")
 android {
   buildFeatures {
     compose = true
+    resValues = false
```
Comment (marianeum) [inline backpack-compose/build.gradle.kts:34]: no resources in the compose module either

File: Backpack/src/main/java/net/skyscanner/backpack/toast/BpkToast.kt (hunk 1 above)
Comment (frugoman) [reply Backpack/src/main/java/net/skyscanner/backpack/toast/BpkToast.kt:35]: is it worth deprecating this api and establishing a new kotlin friendly one? not blocking for this pr necesarily tho

**Review by frugoman — APPROVED** [review]
🚢

File: Backpack/src/main/java/net/skyscanner/backpack/toast/BpkToast.kt (hunk 1 above)
Comment (marianeum) [reply Backpack/src/main/java/net/skyscanner/backpack/toast/BpkToast.kt:35]: so what this does is make sure that java can also still access it correctly without it being very awkward. doesn't mean it's _not_ kotlin friendly.

**Pull request merged**